draft-ietf-acme-scoped-dns-challenges.mkd

```diff
@@ -226,12 +226,12 @@ A client can fulfill this challenge by performing the following steps:
   }
   ~~~
 
-On receiving a response, the server constructs and stores the key authorization from the challenge `token` value and the current client account key.
+On receiving this response, the server validates the JWS and constructs and stores the key authorization from the challenge `token` value and the current client account key.
```

Comment:
Can we say "validates the message" or "validates the JWS and message"?

There is more to validate than just the JWS. (Verify the challenge is associated with given account/kid, etc)

```suggestion
On receiving this response, the server validates the message and constructs and stores the key authorization from the challenge `token` value and the current client account key.
```

Reply:
This sounds good to me!

Reply:
Are we good to merge this now?